global_helpers/panther_greynoise_helpers.py

```diff
@@ -0,0 +1,155 @@
+import ast
+import datetime
+
+from dateutil import parser
+from panther_base_helpers import PantherUnexpectedAlert, deep_get
+
+
+class GreyNoiseBasic:
+    def __init__(self, event):
+        self.noise = deep_get(event, "p_enrichment", "greynoise_noise_basic")
+        if self.noise is None:
+            self.advanced_enabled = deep_get(event, "p_enrichment", "greynoise_noise_advanced")
+            if self.advanced_enabled is not None:
+                raise PantherUnexpectedAlert(
+                    "This account is configured with an Advanced GreyNoise Subscription.\
+                     Please use GreyNoiseAdvanced"
```

Comment:
All exception text is currently just a placeholder. Need to validate verbiage with product

Reply:
can we make the text constants so they are easy to change?

Reply:
Yeah I think this probably should become its own  Exception Class honestly. I kinda just snagged this one because it already existed